### PR TITLE
Updated readme to match updated readme template.

### PR DESCRIPTION
**Pull request by mary-vandyke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,31 +82,28 @@ export PROXY_PASS_APICC='resolver 10.0.0.2 valid=10s; set $backend "http://examp
...
-## Code.gov Registration Info
+# Contribution
+Please read CONTRIBUTING.md for details on our Code of Conduct, the process for submitting pull requests to us, and how contributions will be released.
 
+# Contact Information
 Agency: DOT
 Short Description: The backend API for the ITS CodeHub repository-sharing website.
 Status: Beta
 Tags: transportation, connected vehicles, intelligent transportation systems, javascript, api
 Labor Hours: 0
```
Comment (nickprice-bah) [inline README.md:104]: Were these copied in by mistake?

File: README.md [hunk 2]
```diff
@@ -85,31 +82,28 @@ export PROXY_PASS_APICC='resolver 10.0.0.2 valid=10s; set $backend "http://examp
...
-## License
+- Change location of Insight page from main menu to a submenu under Resources.
+- Rename Insight page to Metrics.
 
+# License
 This project is licensed under the Apache 2.0 License.
 
-## Code.gov Registration Info
+# Contribution
+Please read CONTRIBUTING.md for details on our Code of Conduct, the process for submitting pull requests to us, and how contributions will be released.
```
Comment (nickprice-bah) [inline README.md:97]: This should include a hyperlink to the CONTRIBUTING.md file

File: README.md [hunk 3]
```diff
@@ -85,31 +82,28 @@ export PROXY_PASS_APICC='resolver 10.0.0.2 valid=10s; set $backend "http://examp
...
-The Developers of ITS CodeHub.
+# Version History and Retention
+Changes are in reference with the previous version (3.0)
 
-## License
+- Change location of Insight page from main menu to a submenu under Resources.
+- Rename Insight page to Metrics.
 
+# License
 This project is licensed under the Apache 2.0 License.
```
Comment (nickprice-bah) [inline README.md:94]: This should include a hyperlink to the LICENSE file

File: README.md [hunk 4]
```diff
@@ -85,31 +82,28 @@ export PROXY_PASS_APICC='resolver 10.0.0.2 valid=10s; set $backend "http://examp
...
-
-## Authors
-
-The Developers of ITS CodeHub.
+# Version History and Retention
+Changes are in reference with the previous version (3.0)
 
-## License
+- Change location of Insight page from main menu to a submenu under Resources.
+- Rename Insight page to Metrics.
```
Comment (nickprice-bah) [inline README.md:91]: This should include a hyperlink to a CHANGELOG.md file.

File: README.md [hunk 5]
```diff
@@ -85,31 +82,28 @@ export PROXY_PASS_APICC='resolver 10.0.0.2 valid=10s; set $backend "http://examp
...
 
-## Built With
-
-* [Docker](https://www.docker.com/) - Containerization Platform
-* [Node](https://nodejs.org/en/) - JavaScript Runtime Environment
-* [Aurelia](https://aurelia.io/) - JavaScript Web Application Framework
-
-## Contributing
-
-Please contribute to this repository using GitHub Pull Requests and GitHub Issues.
```
Comment (nickprice-bah) [inline README.md:97]: This should include a hyperlink to a CONTRIBUTING.md file

File: README.md [hunk 6]
```diff
@@ -3,25 +3,22 @@
 # codehub-ui
 > version: 3.2
```
Comment (nickprice-bah) [inline README.md:5]: Not a requirement of the README template, but consider adding Codebuild badge?

**Review by nickpricedev — CHANGES_REQUESTED** [review]
A few formatting changes requested. Overall content looks good.

**Review by nickpricedev — APPROVED** [review]

**Pull request merged**